internal/logtail/logtail.go

```diff
@@ -64,7 +64,11 @@ func NewLogTail(ctx context.Context, logFile string) (chan string, chan error, e
 
 				switch event.Op {
 				case fsnotify.Write:
-					peekAndRead(f, reader, &logs, &errs)
+					for {
```

Comment:
We can get multi-line writes with one event triggered. 
IMO something better would be for `peekAndRead` to look ahead for the last `newline` so we end up with a single write.